src/REAXFF/fix_qeq_rel_reaxff.cpp

```diff
@@ -0,0 +1,112 @@
+
+#include "fix_qeq_rel_reaxff.h"
+
+#include "atom.h"
+#include "comm.h"
+#include "error.h"
+#include "fix_efield.h"
+#include "force.h"
+#include "neighbor.h"
+
+#include "pair_reaxff.h"
+#include "reaxff_api.h"
+
+#include <cmath>
+#include <cstring>
+
+using namespace LAMMPS_NS;
+using namespace FixConst;
+
+static constexpr double ANGSTROM_TO_BOHRRADIUS = 1.8897261259;
+
+/* ---------------------------------------------------------------------- */
+
+FixQEqRelReaxFF::FixQEqRelReaxFF(LAMMPS *lmp, int narg, char **arg) : FixQtpieReaxFF(lmp, narg, arg)
+{
+}
+
+/* ---------------------------------------------------------------------- */
+
+void FixQEqRelReaxFF::calc_chi_eff()
+{
+  memset(&chi_eff[0], 0, atom->nmax * sizeof(double));
+
+  const auto x = (const double *const *) atom->x;
+  const int *type = atom->type;
+
+  double dist, overlap, sum_n, sum_d, expa, expb, chia, phia, phib, p, m;
+  int i, j;
+
+  // check ghost atoms are stored up to the distance cutoff for overlap integrals
+  const double comm_cutoff = MAX(neighbor->cutneighmax, comm->cutghostuser);
+  if (comm_cutoff < dist_cutoff / ANGSTROM_TO_BOHRRADIUS) {
+    error->all(FLERR, Error::NOLASTLINE,
+               "Comm cutoff {} is smaller than distance cutoff {} for overlap integrals in fix {}. "
+               "Increase accordingly using comm_modify cutoff",
+               comm_cutoff, dist_cutoff / ANGSTROM_TO_BOHRRADIUS, style);
+  }
+
+  // efield energy is in real units of kcal/mol, factor needed for conversion to eV
+  const double qe2f = force->qe2f;
+  const double factor = 1.0 / qe2f;
+
+  if (efield) {
+    if (efield->varflag != FixEfield::CONSTANT) efield->update_efield_variables();
+
+    // compute chi_eff for each local atom
+    for (i = 0; i < nn; i++) {
+      expa = gauss_exp[type[i]];
+      chia = chi[type[i]];
+      if (efield->varflag != FixEfield::ATOM) {
+        phia = -factor * (x[i][0] * efield->ex + x[i][1] * efield->ey + x[i][2] * efield->ez);
+      } else {    // atom-style potential from FixEfield
+        phia = efield->efield[i][3];
+      }
+
+      sum_n = 0.0;
+      sum_d = 0.0;
+
+      for (j = 0; j < nt; j++) {
+        dist = distance(x[i], x[j]) * ANGSTROM_TO_BOHRRADIUS;    // in atomic units
+
+        if (dist < dist_cutoff) {
+          expb = gauss_exp[type[j]];
+
+          // overlap integral of two normalised 1s Gaussian type orbitals
+          p = expa + expb;
+          m = expa * expb / p;
+          overlap = pow((4.0 * m / p), 0.75) * exp(-m * dist * dist);
+
+          if (efield->varflag != FixEfield::ATOM) {
+            phib = -factor * (x[j][0] * efield->ex + x[j][1] * efield->ey + x[j][2] * efield->ez);
+          } else {    // atom-style potential from FixEfield
+            phib = efield->efield[j][3];
+          }
+          sum_n += (chia + scale * (phia - phib)) * overlap;
+          sum_d += overlap;
+        }
+      }
+      chi_eff[i] = sum_n / sum_d;
```

Comment:
```suggestion
          sum_n += scale * phib * overlap;
          sum_d += overlap;
        }
      }
      chi_eff[i] = chia + scale * phia - sum_n / sum_d;
```

I assume this is the same -- it saves a few FLOPs and is more readable (especially in lining up the source code with the docs description)

Reply:
Correct, I've included this change :)